### ISSUE TITLE
MacOS installer signing fails with error code 109

### ISSUE THREAD

**Issue by jaimergp** [napari/napari#4866]
Just writing down that macos PKG signing for the bundles have been failing for a while (a month or so). Need to debug locally. The error code (109) is undocumented. Certificate import does work (previous step in CI) so it might be that:

* Certificate has been revoked so it cannot be used for signing
* Timestamp server is rejecting calls from CI
* Something else?

I'll tackle this in `napari/packaging`.

**Comment by sofroniewn** [comment]
Thanks for flagging @jaimergp - maybe @potating-potato can look in and let me know if he needs my support

**Comment by potating-potato** [comment]
sorry I don't see the related 109 error anywhere in the logs, the logs is saying the build is failing, did I miss something?

**Comment by potating-potato** [comment]
nvm I was looking at a particular run that failed

**Comment by jaimergp** [comment]
Yes, sorry, this is the certificate we use to sign `conda-standalone` so it passes notarization. In the installer this is internally renamed to `conda.exe`, but it's a regular unix executable.

Just to make sure, this is (roughly) the command that is failing:

```
$ codesign --verbose --sign $OUR_APPLICATION_CERTIFICATE_IDENTITY --options runtime --force --entitlements entitlements.plist conda_standalone.exe
```

The `entitlements.plist` contents should be:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
        <key>com.apple.security.cs.allow-dyld-environment-variables</key>
        <true/>
        <key>com.apple.security.cs.allow-jit</key>
        <true/>
        <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
        <true/>
        <key>com.apple.security.cs.disable-executable-page-protection</key>
        <true/>
        <key>com.apple.security.cs.disable-library-validation</key>
        <true/>
</dict>
</plist>
```

`conda_standalone` can be found here, inside the `standalone_conda` directory in these Anaconda.org tarballs:

* [Apple Intel](https://anaconda.org/napari/conda-standalone/4.12a/download/osx-64/conda-standalone-4.12a-h694c41f_10.tar.bz2)
* [Apple Silicon](https://anaconda.org/napari/conda-standalone/4.12a/download/osx-arm64/conda-standalone-4.12a-hf4c2487_9.tar.bz2)

**Comment by potating-potato** [comment]
I downloaded the artifact and the code sign worked locally: 
`
(base) zliu@CZIMACOS3562 ~ % codesign --verbose --sign "Developer ID Application: Chan Zuckerberg Initiative, LLC (7WAT66VT96)" --prefix org.napari --options runtime --force  --entitlement /tmp/entitlement.plist ~/Downloads/conda-standalone-4.12a-hf4c2487_9/standalone_conda/conda.exe 
/Users/zliu/Downloads/conda-standalone-4.12a-hf4c2487_9/standalone_conda/conda.exe: replacing existing signature
/Users/zliu/Downloads/conda-standalone-4.12a-hf4c2487_9/standalone_conda/conda.exe: signed Mach-O thin (arm64) [org.napariconda]`

**Comment by jaimergp** [comment]
Thanks @potating-potato! I guess I'll have to ssh into the CI runner...
